src/code/selectedOp.ts

```diff
@@ -89,10 +90,18 @@ export async function makeSelectedOperation(opID: OpID) {
   // remove old listeners ? old object should never .store
   op.on("update", () => window.map.fire("wasabee:op:change"));
   op.on("blockers", () => window.map.fire("wasabee:crosslinks"));
+  op.on(
```

Comment:
corruption occurs on op loading (it appeared because blockers in earlier version) so it's too late at this point to catch the event